ReactTask/src/redux/formFeatures/formFeaturesOperations.ts

```diff
@@ -39,14 +40,27 @@ const loginForm =
       const { data } = await axios.post<ITokens>('api/auth/login', credentials);
 
       token.set(data.access_token);
-
       dispatch(loginSuccess(data));
 
       const response = await axios.get<IProfileWithData>('/api/auth/profile');
       dispatch(getUserProfileSuccess(response.data));
+      dispatch(
+        alert({
+          show: true,
+          err: false,
+          message: 'Authorization was successful',
```

Comment:
Messages could be moved into constants file.

Reply:
Moved into constants